packages/@aws-cdk/aws-appconfig-alpha/awslint.json

```diff
@@ -26,12 +26,10 @@
         "docs-public-apis:@aws-cdk/aws-appconfig-alpha.IApplication",
 
         "no-unused-type:@aws-cdk/aws-appconfig-alpha.PredefinedDeploymentStrategyId",
-        "ref-via-interface:@aws-cdk/aws-appconfig-alpha.Application.addAgentToEcs.taskDef",
         "props-physical-name:@aws-cdk/aws-appconfig-alpha.ApplicationProps",
         "props-physical-name:@aws-cdk/aws-appconfig-alpha.DeploymentStrategyProps",
         "props-physical-name:@aws-cdk/aws-appconfig-alpha.EnvironmentProps",
         "props-physical-name:@aws-cdk/aws-appconfig-alpha.ExtensionProps",
-        "events-in-interface",
```

Comment:
How is this awslint error fixed in this PR?

Reply:
Fixed by adding these methods to the application interface

https://github.com/aws/aws-cdk/pull/28671/files#diff-95c038ca91ae8a09cd48e458bab9f54f352b47fc300fda6502045c1fbf4e26fb